File: LEVEL1/문자열을 정수로 바꾸기.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        boolean chk = true;
        int res = 0;
        
        for(int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            
            if(c == '-') {
                chk = false;
            }
            
            else if(c != '+') {
                res = res * 10 + (c - '0');
            }
        }
        
        if(chk) {
            answer = res;
        }
        
        else {
            answer = -1 * res; 
        }
        
        return answer;
    }
}

File: LEVEL1/문자열을 정수로 바꾸기.cpp
#include<string>
#include<vector>
using namespace std;

int solution(string s) {
    int answer = 0, res = 0;
    bool chk = true;
    
    for(int i = 0; i < s.length(); i++) {
        if(s[i] == '-') {
            chk = false;
        }
        
        else if(s[i] != '+') {
            res = res * 10 + (s[i] - '0');
        }
    }
    
    if(chk) {
        answer = res;
    }
    
    else {
        answer = res * -1;
    }
    
    return answer;
}